tests/storage/zfsvol/test_zfsvol_sr.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import time
+import pytest
+
+from lib.commands import SSHCommandFailed
+from lib.common import wait_for, vm_image
+from tests.storage import vdi_is_open
+
+# Requirements:
+# - one XCP-ng host >= 8.2 with an additional unused disk for the SR
```

Comment:
Same comment as on the dependent "more tests" PR: minimum XCP-ng host version should be 8.3. And thus the whole tests should be requiring the `host_at_least_8_3` fixture:

```
@pytest.mark.usefixtures("host_at_least_8_3")
```

Reply:
If SR.forget is a success then PBD objects are removed and doesn't exist anymore.

Reply:
already noted above :)